### ISSUE TITLE
[Bug] Drag Ruler integration fails

### ISSUE THREAD

**Issue by SirMotte** [Roger92/metric-ruler-labels#14]
Drag Ruler integration seems to no longer work. As I understand v10 support of drag ruler saw some underlying API changes happen that his module probably ties into. I'd be glad to see a fix for this. Thank you for this little gem of a module.

**Comment by Roger92** [comment]
Hey, yep I waited for manuel to update drag ruler. That happened this week, but I had not much time to apply changes ... Hope I have time this week. I also want to have it working with drag ruler, because I use dragruler too :)

Nice to know that my module helps you :)

**Comment by SirMotte** [comment]
> Hey, yep I waited for manuel to update drag ruler. That happened this week, but I had not much time to apply changes ... Hope I have time this week. I also want to have it working with drag ruler, because I use dragruler too :)
> 
> Nice to know that my module helps you :)

Glad to hear that. It's small tools like this that make my life as a DM just so much easier. In combination with [this Module](https://github.com/HadaIonut/Foundry-mgl) I don't have to worry about anything metric related anymore AND I can insert a simple field counter as well, that tells me and my players, that they currently are measuring 10 squares or the like, Or I can apply a value for my world map, that tells my not only the distance, but also the time needed. awesome!!!!

In short, this module is incredibly felxible and therefore mandatory for my gameplay...on that note I think I need to donate now.

**Comment by Roger92** [comment]
Thx alot, but you don't HAVE to donate 😁

**Comment by SirMotte** [comment]
> Thx alot, but you don't HAVE to donate 😁

I try to do so with all extension I use, so there you go :)

**Comment by Roger92** [comment]
@SirMotte  aaaaalright... had some time today and i think i got it :D Will do some testing tomorrow and then there might be a new version tomorrow :) Sorry for the long wait.

**Comment by SirMotte** [comment]
Please don't apologize! Happy to have your module in working order with drag ruler. Such a useful tool in an of itself, and now even better again.

**Comment by Roger92** [comment]
Alright, Version 2.1.2 is available. If you see any bugs feel free to report them :)

**Comment by SirMotte** [comment]
> Alright, Version 2.1.2 is available. If you see any bugs feel free to report them :)

Currently occupied with world building, will test in the next 2 hours :)

**Comment by Roger92** [comment]
yah no problem ;) Just wanted to let you know :)

**Comment by SirMotte** [comment]
I can confirm the issue has been fixed. Thank you very much!
